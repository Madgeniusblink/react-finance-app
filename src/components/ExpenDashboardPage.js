import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'

const ExpenDashboardPage = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export { ExpenDashboardPage as default }