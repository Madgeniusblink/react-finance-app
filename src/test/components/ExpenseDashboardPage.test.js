// react-test-renderer
import React from 'react'
import { shallow } from 'enzyme'
import ExpenDashboardPage from '../../components/ExpenDashboardPage'

test('should render Expense Dashboard Board Page', () => {
    const wrapper = shallow(<ExpenDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})
