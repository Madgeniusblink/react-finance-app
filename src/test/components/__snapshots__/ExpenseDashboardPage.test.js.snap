// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Dashboard Board Page 1`] = `
<div>
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
